readme.txt

```diff
@@ -1 +1,21 @@
-Ninslash.com
\ No newline at end of file
+Ninslash.com
+
+# Compilation (Ubuntu) #
+
+## Needed libraries ##
+
+```
+sudo apt-get install python alsa (asound) gl glu x11 libsdl freetype bam
```

Comment:
I know you have taken all these from the teeworlds wiki page on compilation, but there it only tells you _what_ to install; that's a totally different thing than 'which packages to install'!
Btw, this wiki page gets outdated more and more as time goes by, and it left linux users pretty much alone with what to do ever since. Figuring it out is not that hard though.

- `python`: will install python 2.7 on most systems, but we need python 3.x → install the `python3` package
- `alsa`: just... wtf?
- `(asound)`: this doesn't belong here
- `gl` `glu` `x11`: all this stuff is named differently
- `libsdl`: Ninslash uses SDL2, and we need the development package → `libsdl2-dev`
- `freetype`: we need the development package, and also it's named differently → `libfreetype6-dev`
- `bam`: great, the only correct one…

Fortunately, there is a command that will install all the dependencies we need to build teeworlds (vanilla). In addition to that, we need to install `libsdl2-dev` and `libglew-dev` for Ninslash. This breaks it down to these two simple commands:

```
sudo apt-get build-dep teeworlds
sudo apt-get install libsdl2-dev libglew-dev
```